include/cantera/kinetics/InterfaceKinetics.h

```diff
@@ -287,6 +287,17 @@ class InterfaceKinetics : public Kinetics
      */
     int phaseStability(const size_t iphase) const;
 
+    //! Gets the interface current for the ith phaseExistence
+    /*!
+    * @param iphase Phase Id
+    * @return The double specifying the interface current. The interface Current
+    *         is useful when charge transfer reactions occur at an interface. It
+    *         is defined here as the net positive charge entering the phase
+    *         specified by the Phase Id. (Units: A/m^2 for a surface reaction,
+    *         A/m for an edge reaction).
+    */
+    double InterfaceCurrent(const size_t iphase);
```

Comment:
```suggestion
    double interfaceCurrent(const size_t iphase);
```
methods should be lower case.

Reply:
Thank you for all the comments. I updated the method to be lower case everywhere that it is referenced.